charts/akeyless-api-gateway/templates/deployment.yaml

```diff
@@ -144,6 +151,29 @@ spec:
             - name: CONFIG_PROTECTION_KEY_NAME
               value: {{ .Values.akeylessUserAuth.configProtectionKeyName }}
           {{- end }}
+          {{- if eq (include "akeyless-api-gw.adminAccessUidExist" .) "true"  }}
```

Comment:
Put both of the env vars ADMIN_UID_TOKEN and GATEWAY_CLUSTER_CACHE in one if statement of {{- if eq (include "akeyless-api-gw.adminAccessUidExist" .) "true" }}

line 114